testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/resource/SseBroadcastResource.java

```diff
@@ -15,18 +14,17 @@
 import javax.ws.rs.sse.SseEventSink;
 import java.io.IOException;
 
-@Singleton
 @Path("/broadcast")
 public class SseBroadcastResource {
 
-   private final Object sseBroadcasterLock = new Object();
-   private volatile SseBroadcaster sseBroadcaster;
-   private volatile boolean onErrorCalled = false;
-   private volatile boolean onCloseCalled = false;
+   private static final Object sseBroadcasterLock = new Object();
+   private static volatile SseBroadcaster sseBroadcaster;
+   private static volatile boolean onErrorCalled = false;
+   private static volatile boolean onCloseCalled = false;
 
    private static final Logger logger = Logger.getLogger(SseBroadcastResource.class);
 
-   SseEventSink subscribedSink = null;
+   static SseEventSink subscribedSink = null;
```

Comment:
These could be correct or not (I haven't checked), but are they really related to this PR / RESTEASY-2686?

Reply:
The point here is that there is no EJB galleon layer ready. But this test was originally using @Singleton annotation, see https://github.com/resteasy/Resteasy/blob/3.12/testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/resource/SseBroadcastResource.java#L18

So basically we have a few choices here:

- ignore this test now and unignore it once EJB layer is ready
- remove Singleton annotation and replace "singleton" variables by static variables
- use ApplicationScope CDI bean instead of  (I don't test it, but it should probably works)

I chose the second option, but I can also try some another, I don't have strong feelings here ...